scripts/launch-xrd

```diff
@@ -250,18 +241,15 @@ while [[ $# -gt 0 ]]; do
            shift
            ;;
         *)
-            if [[ ! $IMAGE ]]; then
-                IMAGE=$1
-                shift
-            else
-                echo "Unexpected argument '$1'" >&2
-                bad_usage
-            fi
-            ;;
+           CTR_MGR_ARGS+=("$1")
+           shift
+           ;;
     esac
 done
 
-if [[ ! $IMAGE ]]; then
+if [[ ! $IMAGE && -n ${CTR_MGR_ARGS[-1]} ]]; then
```

Comment:
AFAICT the `IMAGE` variable will never be set here? And so also the 'elif' branch should be an 'else' branch?
```suggestion
if [[ ${CTR_MGR_ARGS[-1]} ]]; then
```